src/metatensor/models/utils/neighbor_list.py

```diff
@@ -0,0 +1,103 @@
+from typing import List
+
+import ase
+import numpy as np
+import torch
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import (
+    NeighborsListOptions,
+    System,
+    register_autograd_neighbors,
+)
+
+
+def calculate_neighbor_lists(
+    system: System, neighbor_lists: List[NeighborsListOptions]
+) -> System:
+    """Calculate neighbor lists for a `System` object.
+
+    :param system: The system for which to calculate neighbor lists.
+    :param neighbor_lists: A list of `NeighborsListOptions` objects,
+        each of which specifies the parameters for a neighbor list.
+
+    :return: The `System` object with the neighbor lists added.
+    """
+    # Convert the system to an ASE atoms object
+    positions = system.positions.detach().cpu().numpy()
+    numbers = system.species.detach().cpu().numpy()
+    cell = system.cell.detach().cpu().numpy()
+    pbc = (
```

Comment:
As an option:

```py
pbc = cell.any(axis=1)
```

Reply:
I've just looked into how we convert Systems to ASE:
https://github.com/Luthaf/rascaline/blob/f00d4406763aa5022ead75a08f2e266e005c81eb/python/rascaline/rascaline/systems/ase.py#L56-L67
For now, we don't support different PBCs for different cell vectors so it won't make a difference. I'll go with your suggestion which looks a bit more elegant as well. Pinging @Luthaf for confirmation

Reply:
Sure, got it

Reply:
This code is no longer there, what was the question?

Reply:
It was about defining the pbc for an `ase.Atoms` object. We discussed if this